grc/blocks/options.block.yml

```diff
@@ -51,7 +51,7 @@ parameters:
 -   id: cmake_opt
     label: CMake options
     dtype: string
-    default: ''
+    default: '-DCMAKE_CXX_STANDARD=11'
```

Comment:
Add -std=c++11 to default CMake options as gnuradio requires C++ >= 11

Reply:
I wonder if DCMAKE_CXX_STANDARD=11 option should go in the CMakeLists.txt.mako file instead?

Reply:
Yeah I think it should.